packages/scandipwa/src/component/MyAccountAddressPopup/MyAccountAddressPopup.container.js

```diff
@@ -112,6 +113,21 @@ export class MyAccountAddressPopupContainer extends PureComponent {
         return this.handleCreateAddress(address);
     }
 
+    showNotification(status, operation) {
+        const { showSuccessNotification, showErrorNotification } = this.props;
+        const message = `Your ${operation} the address`;
```

Comment:
Please make sure all text you display on the page is translatable.
You can use this example:
```javascript
const message = __('Your %s  the address', operation);
```

Reply:
I believe `Your` has a typo? Did you mean `You`?

Reply:
Both are fixed